crates/release_plz/src/args/update.rs

```diff
@@ -19,8 +19,14 @@ pub struct Update {
     /// Path to the Cargo.toml of the project you want to update.
     /// If not provided, release-plz will use the Cargo.toml of the current directory.
     /// Both Cargo workspaces and single packages are supported.
-    #[arg(long, value_parser = PathBufValueParser::new())]
+    // #[arg(long, value_parser = PathBufValueParser::new())]
     project_manifest: Option<PathBuf>,
+
+    /// Path to the Cargo.toml of the project you want to update.
+    /// If not provided, release-plz will use the Cargo.toml of the current directory.
+    /// Both Cargo workspaces and single packages are supported.
+    #[arg(long, value_parser = PathBufValueParser::new())]
```

Comment:
```suggestion
    #[arg(long, value_parser = PathBufValueParser::new(), alias = "project-manifest")]
```
With a clap [alias](https://docs.rs/clap/latest/clap/struct.Arg.html#method.alias), you can delete the `project_manifest` variable 👍

Reply:
Thanks for the suggestion! I will do that and submit it.

Reply:
Hi @MarcoIeni, I implemented your suggestion. It should reflect in the latest commits. 
All tests are passing. Code formatting done as per Contributing.MD

Please have a look when you can. Cheers! 😄